lib/rucio/gateway/account_limit.py

```diff
@@ -56,16 +66,22 @@ def get_local_account_limits(account, vo='def', *, session: "Session"):
     :returns: The account limits.
     """
 
-    account = InternalAccount(account, vo=vo)
+    internal_account = InternalAccount(account, vo=vo)
```

Comment:
This rename should probably be a different commit (same goes for lines 98, 123, 152 etc)

Reply:
It's directly related to the type annotation change (because by specifying `account: str`, the linter complains about this line as we're now passing an `InternalAccount` to an object that should be `str`), but I'll break these out in a separate commit for clarity

Reply:
Addressed in latest forcepush